scripts/src/java/org/oppia/android/scripts/regex/RegexPatternValidationCheck.kt

```diff
@@ -51,6 +58,13 @@ fun main(vararg args: String) {
       isFailing || checkFailed
     }
 
+  if (hasFileContentCheckFailure) {
+    println(
+      "Refer to https://github.com/oppia/oppia-android/wiki/Static-Analysis-Checks for more" +
+        " details on how to fix this.\n"
+    )
+  }
```

Comment:
We should set up this error such that we only need to print this once. Ditto for other checks.

Reply:
Done.